CONTRIBUTING.md

```diff
@@ -75,3 +69,37 @@ Finally, please limit your pull requests to contain only one feature at a time.
 ## Getting Help
 
 If you want to talk with other folks in the Devopness community (including members of the Devopness team) please [start a discussion](https://github.com/devopness/devopness/discussions) and we will soon get in touch with you.
+
+## Releases
+
+We publish package releases and changelogs using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github) together with Changesets.
```

Comment:
I don't think here the reader of this file needs help to find out the docs for `GitHub releases`. Perhaps could be more useful to guide them to see the history of releases i this repo?

```suggestion
We publish package releases and changelogs using [GitHub Releases](https://github.com/devopness/devopness/releases) together with Changesets.
```